.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
```

Comment:
You set the Node version on both `.nvmrc` and `.tool-versions`, would you want to set the version to use for Travis CI too? I feel like now it will always run with the latest stable.

Reply:
Travis CI will use `.nvmrc` for the version: https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions-using-nvmrc

But perhaps I should run tests against multiple versions of node, depending on which versions we want to support

Reply:
That's cool - didn't know Travis does that.

Not sure if you need multiple Node versions - it is only used for building.